grails-app/conf/Config.groovy

```diff
@@ -86,7 +86,7 @@ downloadAuth {
 }
 
 featureToggles {
-    // someFeature = <boolean>
+    dynamicGeoserverFilters = false
```

Comment:
Do we maybe need to have a configuration per server saying whether or not it has this plugin installed? That way we'd only send requests to servers we know can respond to them.